packages/router/src/__tests__/set.test.tsx

```diff
@@ -10,14 +10,18 @@ import { Set } from '../Set'
 const ChildA = () => <h1>ChildA</h1>
 const ChildB = () => <h1>ChildB</h1>
 const ChildC = () => <h1>ChildC</h1>
-const GlobalLayout: React.FC = ({ children }) => (
+const GlobalLayout: React.FunctionComponent<PropsWithChildren> = ({
```

Comment:
Personally I'd rather not use `PropsWithChildren`. Dan himself has said that they mainly added `PropsWithChildren` to make it easier for them to write codemods. He instead recommends to manually add `chilren` to your Props interface, which I also like more. https://stackoverflow.com/a/71809927/88106